src/schemas/transaction.json

```diff
@@ -45,100 +208,64 @@
 			}
 		]
 	},
-	"TransactionWithSender": {
-		"title": "Transaction object with sender",
+	"Transaction2930Signed": {
+		"title": "Signed 2930 Transaction",
 		"type": "object",
 		"allOf": [
+			{ "$ref": "#/components/schemas/Transaction2930Unsigned"},
 			{
-				"required": ["from"],
+				"title": "EIP-2930 transaction signature properties.",
+				"required": [ "yParity", "r", "s" ],
 				"properties": {
-					"from": {
-						"title": "from",
-						"$ref": "#/components/schemas/address"
+					"yParity": {
+						"title": "yParity",
+						"description": "The parity (0 for even, 1 for odd) of the y-value of the secp256k1 signature.",
+						"$ref": "#/components/schemas/uint"
+					},
+					"r": {
+						"title": "r",
+						"$ref": "#/components/schemas/uint"
+					},
+					"s": {
+						"title": "s",
+						"$ref": "#/components/schemas/uint"
 					}
 				}
-			},
-			{ "$ref": "#/components/schemas/Transaction" }
+			}
 		]
 	},
-	"Transaction": {
+	"TransactionLegacySigned": {
+		"title": "Signed Legacy Transaction",
 		"type": "object",
-		"title": "Transaction object",
 		"allOf": [
+			{ "$ref": "#/components/schemas/TransactionLegacyUnsigned"},
 			{
-				"required": ["nonce", "gas", "value", "input"],
+				"title": "Legacy transaction signature properties.",
+				"required": [ "yParity", "r", "s" ],
```

Comment:
Is `yParity` also supposed to be here? I think `v` is more appropriate for legacy txs since the chain id may also be encoded.

Reply:
Nope, just a typo.  Will fix when next at a computer.

Reply:
Don't worry about it, I will take care of it.